disperser/cmd/batcher/main.go

```diff
@@ -84,6 +84,10 @@ func RunBatcher(ctx *cli.Context) error {
 		return err
 	}
 
+	// TODO remove this log statement
```

Comment:
Maybe we could leave it as trace or debug?

Reply:
Set to debug level.